modules/gitops-release-notes-1-16-0.adoc

```diff
@@ -96,6 +96,14 @@ With this update, this issue is fixed. This update resolves the issue by integra
 [id="known-issues-1-16-0_{context}"]
 == Known Issues
 
-* There is currently a known issue that assigns a lower `SecurityContextConstraints` (SCC) to Redis service account in {gitops-shortname} v1.16 after upgrading from {gitops-shortname} v1.15 to v1.16. The {gitops-shortname} Operator does not update the `securityContext` of the `redis-ha-server` StatefulSet, which causes the container's user to be statically set instead of being randomly assigned as required by the `restricted-v2` SCC. The `redis-ha-server` StatefulSet pods retain the old configurations and are not updated with the new settings. As a result, the new configuration of the StatefulSet is not applied correctly.
+* There is currently a known issue that assigns a lower `SecurityContextConstraints` (SCC) to Redis service account in {gitops-shortname} v1.16 after upgrading from {gitops-shortname} v1.15 to v1.16. The {gitops-shortname} Operator does not update the `securityContext` of the `redis-ha-server` `StatefulSet`, which causes the container's user to be statically set instead of being randomly assigned as required by the `restricted-v2` SCC. The `redis-ha-server` `StatefulSet` pods retain the old configurations and are not updated with the new settings. As a result, the new configuration of the `StatefulSet` is not applied correctly.
 +
-Workaround: Manually delete the `redis-ha-server` StatefulSet to trigger the re-creation of the pods with the updated settings. link:https://issues.redhat.com/browse/GITOPS-6670[GITOPS-6670]
\ No newline at end of file
+Workaround: Manually delete the `redis-ha-server` `StatefulSet` to trigger the re-creation of the pods with the updated settings. link:https://issues.redhat.com/browse/GITOPS-6670[GITOPS-6670]
+
+[id="deprecated-features-1-16-0_{context}"]
+== Deprecated and removed features
+
+[id="deprecation-of-some-fields-in-argo-cd_{context}"]
+=== Deprecation of `.spec.initialRepositories` & `.spec.repositoryCredentials` fields in Argo CD
+
+* In {gitops-title} v1.16, the `.spec.initialRepositories` and `.spec.repositoryCredentials` fields in Argo CD are deprecated and will be removed in a future release. Update your configurations to remove dependencies on these fields, as they will no longer be supported. To add or modify a repository, use the Argo CD web UI or CLI. link:https://issues.redhat.com/browse/GITOPS-5961[GITOPS-5961]
```

Comment:
As written, this sentence is incorrect (fields will not be removed, and `.spec` fields are directly not part of Argo CD). This is a corrected version (@anandrkskd can correct me if wrong):
* In {gitops-title} v1.16, the `.spec.initialRepositories` and `.spec.repositoryCredentials` fields *in the `ArgoCD` CR* are deprecated ~and will be removed in a future release~. *These fields will no longer be supported by  {gitops-title} and Argo CD  in a future release*. Update your configurations to remove dependencies on these fields ~as they will no longer be supported~. To add or modify a repository, use the Argo CD web UI or CLI. link:https://issues.redhat.com/browse/GITOPS-5961[GITOPS-5961]

Reply:
If I understand correctly, we want to remove `.spec.initialRepositories` and `.spec.repositoryCredentials` field from ArogCD CR in upcoming 1.17 release, as per the discussion happened in last  cabal call.

Reply:
@jgwest, please let me know what do you think?

Reply:
Both sentences make sense to me. In 1.17, even if we don't remove the fields, we will drop the  functionality as upstream doesn't support it and eventually remove these fields whenever we do a API version upgrade of the ArgoCD CR in future. 

What do you think about?
> In {gitops-title} v1.16, the `.spec.initialRepositories` and `.spec.repositoryCredentials` fields in the `ArgoCD` CR are deprecated and will be removed in a future release. These fields will no longer be supported by {gitops-title}  in a future release. Update your configurations to remove dependencies on these fields. To add or modify a repository, use the Argo CD web UI or CLI. link:[https://issues.redhat.com/browse/GITOPS-5961[GITOPS-5961]](https://issues.redhat.com/browse/GITOPS-5961%5BGITOPS-5961%5D)

Reply:
Wouldn't it be more confusing for users to have fields but not working?

Reply:
I think it is an accepted practice to keep deprecated fields around without functionality. With proper documentation/ k8s events, we can avoid confusion.

The recommended approach is to remove fields only when performing a CRD API upgrade.

Reply:
In that case, we can go ahead with what Jonathan suggested. 
Thanks, @jgwest @svghadi for the review.

Reply:
Hello @jgwest @anandrkskd @svghadi and @varshab1210, after the end of the discussion in this thread, I have updated this release note by taking care of your suggestions. Please let me know if the content looks fine.